.github/workflows/COMMENT_DEVELOP_FIX.yml

```
@@ -0,0 +1,32 @@
+name: COMMENT_DEVELOP_FIX
+on:
+  pull_request:
+    branches:
+      - develop
+permissions:
+  pull-requests: write
+jobs:
+  comment-on-fix-to-develop:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for fix commits
+        env:
+          COMMITS_URL: ${{ github.event.pull_request.commits_url }}
+        run: |
+          echo "Checking commits via $COMMITS_URL"
+
+          if [[ $(curl $COMMITS_URL) =~ ."\"message\": \"fix".* ]]
+          then
+            echo "FIX_COMMITS_PRESENT=true" >> $GITHUB_ENV
+          else
+            echo "FIX_COMMITS_PRESENT=false" >> $GITHUB_ENV
+          fi
+      - name: Create comment
+        if: ${{ env.FIX_COMMITS_PRESENT == 'true' }}
+        uses: peter-evans/create-or-update-comment@v2
```

Let's pin the 3rd party action to a commit to prevent any malicious action. Cf. https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

```suggestion
        uses: peter-evans/create-or-update-comment@c9fcb64660bc90ec1cc535646af190c992007c32
```

---

The commit I suggest pinning is the current v2: https://github.com/peter-evans/create-or-update-comment/commit/c9fcb64660bc90ec1cc535646af190c992007c32

---

Ah, that makes a lot of sense. Thanks for hinting this, I did not think about that